### ISSUE TITLE
Need a simple way to set working directory for Run and Debug

### ISSUE THREAD

**Issue by swpalmer** [kelemen/netbeans-gradle-project#60]
It should be trivial to configure the working directory to use when running your code.

It should be possible to set this as a Gradle project property, much like the mainClass, or in the project property dialog for the built-in tasks.

**Comment by eino-makitalo** [comment]
In Java / Groovy there is no way to change working directory.  Problem is like #18 that netbeans gradle plugin should start "process" in project root directory, but for me it's still using c:\program files\netbeans instead.  When running programs with gradle everything works but when running in netbeans gradle plugin it does not.

**Comment by eino-makitalo** [comment]
I study this further and I think it's too deep to me to fix.  It's using ThreadPoolExecutor and it will allways work differently from real gradle / gradlew?

**Comment by aplatypus** [comment]
Does this method work for Test-s too?

@eino-makitalo The gradle documentation says that test and run workingDir defaults to the project.projectDir.   So the same location as your build.gradle.

**Comment by kelemen** [comment]
@aplatypus yes, because [Test](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html) task also has a `workingDir` property.
